File: mall-sellergoods-interface/src/main/java/com/marshal/sellergoods/service/SellerService.java
package com.marshal.sellergoods.service;


import com.marshal.pojo.TbSeller;
import com.marshal.util.ResponseData;
import java.util.List;

public interface SellerService {
	public List<TbSeller> findAll();

	public ResponseData query(TbSeller condition,int pageNum,int pageSize);

	public TbSeller queryById(String id);

	public void save(TbSeller tbSeller);

	public void delete(String[] ids);
}

File: mall-shop-web/src/main/webapp/js/service/sellerService.js
// service层,与数据库交互
app.service("sellerService",function($http){
    //BASE crud
    this.query=function(pageNum,pageSize,condition){
        return $http.post("../seller/query?pageNum="+pageNum+"&pageSize="+pageSize,condition);
    }

    this.save=function (seller) {
        return $http.post("../seller/save",seller);
    }

    this.remove=function (idList) {
        return $http.get("../seller/delete?idList="+idList);
    }

    this.queryById=function (id) {
        return $http.get("../seller/queryById?id="+id);
    }

    this.register=function (seller) {
        return $http.post("../seller/register",seller);
    }
});